src/tests/utils/debug.cairo

```diff
@@ -0,0 +1,16 @@
+use core::fmt::{Debug, Formatter, Error};
+use starknet::{ContractAddress, ClassHash};
+
+impl DebugContractAddress of core::fmt::Debug<ContractAddress> {
+    fn fmt(self: @ContractAddress, ref f: Formatter) -> Result<(), Error> {
+        let address: felt252 = (*self).into();
+        write!(f, "{address:?}")
+    }
+}
+
+impl DebugClassHash of core::fmt::Debug<ClassHash> {
+    fn fmt(self: @ClassHash, ref f: Formatter) -> Result<(), Error> {
+        let hash: felt252 = (*self).into();
+        write!(f, "{hash:?}")
+    }
+}
```

Comment:
👍

Reply:
Also, should we document this addition in the CHANGELOG?

Reply:
Not sure if we should add test stuff in the changelog. Actually, I think we shouldn't, even when I added a couple of entries already. I think changelog should be for API-related stuff of the library, not including testing, what do you think?

Reply:
Yeah, I agree with not including testing

Reply:
cc @martriay